### PR TITLE
Implement uiElementID, IDforLabel + clean up CSS classes for all components

### LINKED ISSUE #71: Remove redundant CSS classes + implement uuID
`uiElementID` was intorduced via #60 PR for #19 and i think that it's generally good approach. This issue is about standarizing this for all components

### PR DESCRIPTION
**Pull request by allozaur**
Close #71

### CODE REVIEW

File: src/utils/uuid.js [hunk 1]
```diff
@@ -1,11 +1,21 @@
...
+  //   uiElementID() {
+  //     // if (this.id) return this.id
+  //     if (this.baseClassname) {
+  //       return `${this.baseClassname}_${uuid()}`
+  //     } else return `nui_${uuid()}`
+  //   }
+  // },
+  methods: {
+    uiElementID(name) {
+      if (name && this.id) return `${this.id}_${uuid()}`
```
Comment (ibockowsky) [inline src/utils/uuid.js:14]: Shouldn't we use `${name}_${uuid()}` here?

File: src/utils/uuid.js [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
-      }
+  // computed: {
+  //   uiElementID() {
+  //     // if (this.id) return this.id
+  //     if (this.baseClassname) {
+  //       return `${this.baseClassname}_${uuid()}`
+  //     } else return `nui_${uuid()}`
+  //   }
+  // },
+  methods: {
```
Comment (ibockowsky) [inline src/utils/uuid.js:12]: I think we should stay with `computed` instead of using `method` for it

**Review by ibockowsky — CHANGES_REQUESTED** [review]
- [ ] Change method to computed property
- [ ] Check if naming is correct

File: src/utils/uuid.js (hunk 1 above)
Comment (allozaur) [reply src/utils/uuid.js:14]: There are cases in which we need to pass argument for this function and thus we need it as a method.

File: src/utils/uuid.js (hunk 2 above)
Comment (allozaur) [reply src/utils/uuid.js:12]: https://github.com/naked-ui/vue/pull/93#discussion_r651586216

File: src/utils/uuid.js (hunk 1 above)
Comment (allozaur) [reply src/utils/uuid.js]: Fixed it in 98cf3c849d824ddd71c32158a48afd3439a63a57

**Review by ibockowsky — APPROVED** [review]

**Pull request merged**